shipyard-controller/models/sequence_execution_test.go

```diff
@@ -570,6 +571,120 @@ func TestSequenceExecution_CompleteCurrentTask(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
let's add some additional test cases to keep the coverage of `sequence_execution.go` at 100%